spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -527,24 +527,24 @@ def assert_relationship_tree
               }
             }
           },
-          [EmsFolder, "vm", {:is_datacenter => false}]   => {
-            [EmsFolder, "Discovered virtual machine", {:is_datacenter => false}]                    => {},
-            [EmsFolder, "GreggT", {:is_datacenter => false}]                                        => {
+          [EmsFolder, "vm"]   => {
+            [EmsFolder, "Discovered virtual machine"]                                               => {},
+            [EmsFolder, "GreggT"]                                                                   => {
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "DEV-GreggT"]                 => {},
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "GT Nightly Appl 2011-02-19"] => {}
             },
-            [EmsFolder, "GregM", {:is_datacenter => false}]                                         => {
+            [EmsFolder, "GregM"]                                                                    => {
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "DEV-GregM"]                  => {},
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "GM Nightly Appl 2011-05-19"] => {}
             },
-            [EmsFolder, "Harpreet", {:is_datacenter => false}]                                      => {
+            [EmsFolder, "Harpreet"]                                                                 => {
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "DEV-HarpreetK"]                           => {},
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "HK-Dev Cucumber Nightly Appl 2011-05-19"] => {}
             },
-            [EmsFolder, "Jason", {:is_datacenter => false}]                                         => {
+            [EmsFolder, "Jason"]                                                                    => {
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "Jason Nightly Appl 2011-02-19"] => {}
             },
-            [EmsFolder, "JoeR", {:is_datacenter => false}]                                          => {
+            [EmsFolder, "JoeR"]                                                                     => {
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "DEV-JoeR"]                                          => {},
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "JR- Dev Cucumber Nightly Appl 2011-05-19 - backup"] => {},
               [ManageIQ::Providers::Vmware::InfraManager::Vm, "JR-cruisecontrol-sandback-testing-April14_backup"]  => {},
```

Comment:
Wow, some of these vm names bring back some memories.

`EVM3_3_2_32`  :see_no_evil: